SU2_CFD/include/numerics/turbulent/transition/trans_sources.hpp

```diff
@@ -68,6 +191,17 @@ class CSourcePieceWise_TransLM final : public CNumerics {
     /*--- "Allocate" the Jacobian using the static buffer. ---*/
     Jacobian_i[0] = Jacobian_Buffer;
     Jacobian_i[1] = Jacobian_Buffer + 2;
+
+    TransCorrelation = config->GetKind_Trans_Correlation();
+    TurbFamily = TurbModelFamily(config->GetKind_Turb_Model());
+
+    if(TransCorrelation == TURB_TRANS_CORRELATION::DEFAULT && TurbFamily == TURB_FAMILY::SA)
+      TransCorrelation = TURB_TRANS_CORRELATION::MALAN;
+    if(TransCorrelation == TURB_TRANS_CORRELATION::DEFAULT && TurbFamily == TURB_FAMILY::KW)
+      TransCorrelation = TURB_TRANS_CORRELATION::MENTER_LANGTRY;
```

Comment:
Does it make sense to do this in CConfig::Set_Postprocessing? Then there would be no worries in other places of the code.

Reply:
Sure! I'll fix it soon, thank you.